src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Schemas/VariationType.cs

```diff
@@ -0,0 +1,57 @@
+using GraphQL.DataLoader;
+using GraphQL.Types;
+using MediatR;
+using VirtoCommerce.InventoryModule.Core.Model.Search;
+using VirtoCommerce.InventoryModule.Core.Services;
+
+namespace VirtoCommerce.XDigitalCatalog.Schemas
+{
+    public class VariationType : ObjectGraphType<ExpVariation>
+    {
+        public VariationType(
+            IMediator mediator,
+            IDataLoaderContextAccessor dataLoader,
+            IProductInventorySearchService productInventorySearchService
+            )
+        {
+            Field<StringGraphType>(
+                "id",
+                description: "Id of variation.",
+                resolve: context => context.Source.Product.Id
+            );
+
+            Field<StringGraphType>(
+                "code",
+                description: "SKU of variation.",
+                resolve: context => context.Source.Product.Code
+            );
+
+            // TODO: change to connection
+            FieldAsync<AvailabilityDataType>("availabilityData", resolve: async context =>
+            {
+                var product = context.Source.Product;
+
+                var invntorySearch = await productInventorySearchService.SearchProductInventoriesAsync(new ProductInventorySearchCriteria
+                {
+                    ProductId = product.Id
+                });
+
+                return new ExpAvailabilityData
+                {
+                    InventoryAll = invntorySearch.Results,
+                    IsActive = product.IsActive ?? false,
```

Comment:
you need to use the same logic as storefront has  https://github.com/VirtoCommerce/vc-storefront/blob/master/VirtoCommerce.Storefront/Domain/Catalog/CatalogService.cs#L243

see how this  has done for xPurchase https://github.com/VirtoCommerce/vc-module-experience-api/blob/dev/src/XPurchase/VirtoCommerce.XPurchase/Specifications/ProductIsAvailableSpecification.cs